Extract bootstrap to conveyor theme

DanielHall977 opened the pull request:
In order to use the theming, add:

```
@import '../node_modules/@autoinvent/conveyor/css/index';
@import '../node_modules/@autoinvent/conveyor/css/styles.scss';
```

to wherever you have

```
@import '../node_modules/@autoinvent/conveyor/css/index';
```

itechify reviewed (commented):
Looks good. I think it still needs a couple of tweaks though. I think the styles.scss file should ONLY have @extend and they should only be extending bootstrap classes.

For example `.conv-disabled-input` has extra styling in its child divs. That styling should either stay in the inline styling where it previously was, or it can be added to the conveyor index.css file. Additionally, the child divs are extending classes like `.disabled-input-padding` and `.disabled-input-value`. These are conveyor classes so they should not be extended. I think there are also other cases too where non-bootstrap classes are being extended.

If you want to test for these, you can do `npm install -g sass` to install Dart Sass globally, and then run `sass css/styles.scss styles.css` (from the main project directory). This will attempt to transform the scss to css and will run into errors most likely. You will also need to add bootstrap4 as a dev dependency (`yarn add --dev bootstrap@4.6.0`) and uncomment the import at the top of the sass file to get this to work (`@import '../node_modules/bootstrap/scss/bootstrap';`)

DanielHall977 commented:
Cleaned up all of the non-bootstrap classes reference in styles.scss
moved index.css to styling.css and used sass to build a new combined index.css file which we can reference in the future.
(note, somt does not seem to play nice with this new version. I'm creating a ticket to update it)
